__tests__/nc-games.test.js

```diff
@@ -15,26 +15,77 @@ afterAll(() => {
 describe("/api/categories", () => {
     describe("/GET", () => {
 
-        test("Status 200 returned when the GET HTTP method is used on endpoint", () => {
-            return request(app)
-            .get("/api/categories")
-            .expect(200)
-        })
-
         test("Status 200: Returns all categories when the GET HTTP method is used on the categories endpoint", () => {
             return request(app)
             .get("/api/categories")
+            .expect(200)
             .then(({body}) => {
                 const { categories } = body;
                 expect(categories).toBeInstanceOf(Array);
                 expect(categories).toHaveLength(4);
-                categories.forEach(() => {
-                    expect.objectContaining ({
-                        slug: expect.any(String),
-                        description: expect.any(String)
-                    })
+                categories.forEach((category) => {
+                    expect(category).toEqual(
+                        expect.objectContaining ({
+                            slug: expect.any(String),
+                            description: expect.any(String)
+                        })
+                    )
+
+                })
+            })
+        })
+    })
+})
+
+describe("/api/reviews/:review_id", () => {
+
+    const reviewObject = {
+        review_id: expect.any(Number),
```

Comment:
You are making a request for a single review here, so you can get away with hardcoding the review_id to make sure you got the right review back!

Reply:
Good point! Will reamend that. Thanks!